src/syscheckd/fim_diff_changes.c

```diff
@@ -996,17 +968,21 @@ int fim_diff_process_delete_registry(const char *key_name, int arch){
         snprintf(full_path, PATH_MAX, "%s/registry/[x32] %s", DIFF_DIR, encoded_key);
     }
 
-    if(fim_diff_delete_compress_folder(full_path) == -1){
-        return -1;
+    ret = fim_diff_delete_compress_folder(full_path);
+    if(ret == -1){
+        merror(FIM_DIFF_DELETE_DIFF_FOLDER_ERROR, full_path);
+    } else if (ret == -2){
+        mdebug2(FIM_DIFF_FOLDER_NOT_EXIST, full_path);
     }
 
-    return 0;
+    return;
```

Comment:
This return is unnecessary! Because the function has a void return

Reply:
You are right but it is not harmful.